src/test/java/com/kentyou/featurelauncher/cli/FeatureLauncherCliTest.java

```diff
@@ -0,0 +1,501 @@
+		assertEquals("", err.toString());
+	}
+
+	@Test
+	public void testFeatureFileOptionLong() {
+		StringBuilder args = new StringBuilder();
+		args.append("--dry-run");
+		args.append(" ");
+		args.append("--feature-file=");
+		args.append(FEATURE_FILE_PATH);
+
+		int exitCode = new CommandLine(new FeatureLauncherCli()).execute((args.toString().split(" ")));
+		assertEquals(0, exitCode);
+
+		assertTrue(out.toString().contains(String.format("Launching feature %s", FEATURE_FILE_ID)));
+		assertEquals("", err.toString());
+	}
+
+	@Test
+	public void testMissingFeatureFileParameter() {
+		StringBuilder args = new StringBuilder();
+		args.append("--dry-run");
+
+		int exitCode = new CommandLine(new FeatureLauncherCli()).execute((args.toString().split(" ")));
+		assertEquals(2, exitCode);
+
+		assertTrue(err.toString().contains(
+				"Missing required argument (specify one of these): (feature file path | -f=feature file path)"));
+	}
+
+	@Test
+	public void testMissingRequiredParameterForFeatureFileOptionShort() {
+		StringBuilder args = new StringBuilder();
+		args.append("--dry-run");
+		args.append(" ");
+		args.append("-f");
+
+		int exitCode = new CommandLine(new FeatureLauncherCli()).execute((args.toString().split(" ")));
+		assertEquals(2, exitCode);
+
+		assertTrue(
+				err.toString().contains("Missing required parameter for option '--feature-file' (feature file path)"));
+	}
+
+	@Test
+	public void testMissingRequiredParameterForFeatureFileOptionLong() {
+		StringBuilder args = new StringBuilder();
+		args.append("--dry-run");
+		args.append(" ");
+		args.append("--feature-file");
+
+		int exitCode = new CommandLine(new FeatureLauncherCli()).execute((args.toString().split(" ")));
+		assertEquals(2, exitCode);
+
+		assertTrue(
+				err.toString().contains("Missing required parameter for option '--feature-file' (feature file path)"));
+	}
+
+	@Test
+	public void testInvalidValueForFeatureFileOptionShort() {
+		StringBuilder args = new StringBuilder();
+		args.append("--dry-run");
+		args.append(" ");
+		args.append("-f=");
+
+		int exitCode = new CommandLine(new FeatureLauncherCli()).execute((args.toString().split(" ")));
+		assertEquals(2, exitCode);
+
+		assertTrue(err.toString().contains("Invalid value for option '--feature-file'"));
+	}
+
+	@Test
+	public void testInvalidValueForFeatureFileOptionLong() {
+		StringBuilder args = new StringBuilder();
+		args.append("--dry-run");
+		args.append(" ");
+		args.append("--feature-file=");
+
+		int exitCode = new CommandLine(new FeatureLauncherCli()).execute((args.toString().split(" ")));
+		assertEquals(2, exitCode);
+
+		assertTrue(err.toString().contains("Invalid value for option '--feature-file'"));
+	}
+
+	@Test
+	public void testArtifactRepositoryOptionShort() {
+		StringBuilder args = new StringBuilder();
+		args.append("--dry-run");
+		args.append(" ");
+		args.append("-a=");
+		args.append(REMOTE_ARTIFACT_REPOSITORY_URI);
```

Comment:
This isn't the same as the syntax for declaring a repository in the specification chapter.